package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "elm-program-builder",
-  "version": "2.1.4",
+  "version": "2.2.1",
```

Comment:
Strictly speaking this is major change, as there are changes in the API, but no need to change.